community/server/src/docs/ops/security.asciidoc

```diff
@@ -99,7 +101,8 @@ To facilitate domain-specific authorization policies in Neo4j Server, security r
 This makes scenarios like user and role based security and authentication against external lookup services possible.
 See +org.neo4j.server.rest.security.SecurityRule+ in the javadocs downloadable from http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.neo4j.app%22%20AND%20a%3A%22neo4j-server%22[Maven Central (org.neo4j.app:neo4j-server)].
 
-CAUTION: The use of Server Authorization Rules may interact unexpectedly with the built-in <<security-server-auth>>, if enabled.
+[CAUTION]
+The use of Server Authorization Rules may interact unexpectedly with the built-in authentication and authorization (see <<security-server-auth>>), if enabled.
```

Comment:
I would put `if enabled` at the start of the sentence: `If enabled, Server Authorization Rules may interact ...`.